model/job-management/src/main/proto/beam_artifact_api.proto

```diff
@@ -81,7 +81,11 @@ message ProxyManifest {
 }
 
 // A request to get the manifest of a Job.
-message GetManifestRequest {}
+message GetManifestRequest {
+    // (Required) An opaque token representing the entirety of the staged artifacts.
```

Comment:
Reindent and regenerate the Go protos? (sync with Ankur about how to do that; I think Python protos don't need to be regenerated since they're not checked in)

Reply:
Also: this token should be in GetArtifactRequest as well - the token basically identifies the job whose artifacts we're talking about.

I see in the other PR that you've been using the manifest itself for that, but I don't think that's a good idea: in my view, the manifest contains information about artifacts that's independent about how they're stored, e.g. the artifact name in the manifest must NOT be a full path to the file on a distributed filesystem, it must be the original name of the file.

Reply:
Okay, sounds good.  That would be a better arrangement.